File: utils/GrabacionDTO.cs
using System;


namespace UHFAPP.utils
{
    internal class GrabacionDTO
    {
        

        public String orden { get; set; }

        

        public String tag { get; set; }

        public int linea { get; set; }

     


        public GrabacionDTO( string orden, string tag, int linea)
        {
            
            this.orden = orden;
            this.tag = tag;
            this.linea = linea;
          
        }   
        
    }
}

File: src/main/java/com/amipem/mono/clases/GrabacionDTO.java
package com.amipem.mono.clases;

import java.sql.Date;
import java.util.GregorianCalendar;

import javax.persistence.Temporal;

import com.amipem.mono.entidades.Grabacion;
import com.amipem.mono.entidades.Orden;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class GrabacionDTO {
	private int cantidad;

	private String codigo;



	private String tag;
	
	private int linea;
	

}
